README.md

```diff
@@ -5,7 +5,7 @@ Attach pdb to a running Python process.
 ## Install
 
 ```
-python3.8 -mpip install git+https://github.com/jschwinger233/pdbattach.git
+python3.8 -m pip install git+https://github.com/jschwinger233/pdbattach.git
```

Comment:
Any specific reason? As far as I know, `-mpip` abides by the [GNU convention](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html) and [POSIX convention](http://www.iitk.ac.in/esc101/05Aug/tutorial/essential/attributes/_posix.html), both of which stipulates "an option and its argument may or may not appear as separate tokens. (In other words, the whitespace separating them is optional.)".
Since they're equivalent, I am in favor of `-mpip` against `-m pip` merely because it requires less keyboard press...

Reply:
cool, learned that, will close it now